student-store-ui/src/components/ProductDetails/ProductDetails.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect} from "react"
+import { useParams } from "react-router-dom"
+import axios from "axios"
+
+import "./ProductDetails.css"
+
+export default function ProductDetails() {
+  const [productToDisplay, setProductToDisplay] = useState({})
+
+  const { id } = useParams()
+  const storeURL = "http://localhost:3001/store"
```

Comment:
This should be defined outside of the component function, near the top of the file. Ideally this would be defined in a separate file but that's not a big issue for this project.